platooning_control/test/test_pid.cpp

```diff
@@ -3,35 +3,35 @@
 #include <ros/ros.h>
 
 
-TEST(PIDControllerTest, test1)
-{
-    platoon_control::PIDController pid;
-    double res = pid.calculate(40, 38);
-    EXPECT_EQ(-8, res);
-}
+// TEST(PIDControllerTest, test1)
```

Comment:
If we don't need these test cases, let's delete them or disable testing in CMake if necessary.